src/main/java/Ladder_game/Basic_function.java

```diff
@@ -0,0 +1,17 @@
+package Ladder_game;
+
+import java.util.Scanner;
+
+public class Basic_function {
```

Comment:
```suggestion
public class BasicFunction {
```
클래스 이름에 _는 사용하지 않습니다.